lib/postgrex/protocol.ex

```diff
@@ -964,6 +1021,8 @@ defmodule Postgrex.Protocol do
     do: {:commit, nil}
   defp decode_tag("ROLLBACK"),
     do: {:rollback, nil}
+  defp decode_tag( :suspended ),
+    do: {:select, nil}
```

Comment:
Oh maybe not the raw tag, but well need to keep the decoded tag and updated the row count I think?